Week1/solution/solution4.py

```diff
@@ -0,0 +1,13 @@
+"""Write a python script to enter any number, if it is integer number, then check
+its armstrong or not. Print appropriate message if it is not armstrong."""
+n=int(input('enter number :'))
+dul=n
+rev=0
+while n!=0:
+    rem=n%10
+    rev=rev+(rem*rem*rem)
+    n=n//10
+if rev==dul:

```

Comment:
can we use 'is' instead of '==' ? if no then why?

Reply:
Yes we can use "is" instead of "=="